src/codemagic/tools/_app_store_connect/action_groups/builds_action_group.py

```diff
@@ -39,3 +52,137 @@ def get_build_pre_release_version(self, build_id: ResourceId, should_print: bool
             self.api_client.builds.read_pre_release_version,
             should_print,
         )
+
+    @cli.action(
+        'add-beta-test-info',
+        BuildArgument.BUILD_ID_RESOURCE_ID,
+        BuildArgument.BETA_BUILD_LOCALIZATIONS,
+        BuildArgument.LOCALE_DEFAULT,
+        BuildArgument.WHATS_NEW,
+        action_group=AppStoreConnectActionGroup.BUILDS)
+    def add_beta_test_info(self,
+                           build_id: ResourceId,
+                           beta_build_localizations: Optional[Types.BetaBuildLocalizations] = None,
+                           locale: Optional[Locale] = None,
+                           whats_new: Optional[Types.WhatsNewArgument] = None):
+        """
+        Add localized What's new (what to test) information
+        """
+
+        beta_test_info_items = [*beta_build_localizations.value] if beta_build_localizations else []
+
+        if whats_new:
+            beta_test_info_items.append(BetaBuildInfo(whats_new=whats_new.value, locale=locale))
+
+        for item in beta_test_info_items:
+            self.create_beta_build_localization(build_id=build_id, locale=item.locale, whats_new=item.whats_new)
+
+    @cli.action(
+        'submit-to-testflight',
+        BuildArgument.BUILD_ID_RESOURCE_ID,
+        action_group=AppStoreConnectActionGroup.BUILDS)
+    def submit_to_testflight(
+            self, build_id: ResourceId, max_build_processing_wait: Optional[Types.MaxBuildProcessingWait] = None):
+        """
+        Submit build to TestFlight
+        """
+
+        # Workaround to support overriding default value by environment variable.
+        if max_build_processing_wait:
+            max_processing_minutes = max_build_processing_wait.value
+        else:
+            max_processing_minutes = Types.MaxBuildProcessingWait.default_value
+
+        app = self.api_client.builds.read_app(build_id)
+        build = self.api_client.builds.read(build_id)
+
+        self._assert_app_has_testflight_information(app)
+        build = self._wait_until_build_is_processed(build, max_processing_minutes)
```

Comment:
In case we don't need to wait for the processing while using it as a Python API and not from CLI, perhaps it would be nice to add option to omit the waiting. Even if the build is already processed, this will add at least one extra line of log to the output that is going to be excess.

Reply:
It's tempting to avoid waiting if `max_processing_minutes` is false-ish (e.g. `null`, or `0`). Wondering if this approach would backfire

Reply:
If `max_processing_minutes` is `None`, then it will fall back to the default value. So plain falsy-truthy check is not optimal.